homeassistant/components/sharkiq/vacuum.py

```diff
@@ -198,6 +221,35 @@ async def async_locate(self, **kwargs: Any) -> None:
         """Cause the device to generate a loud chirp."""
         await self.sharkiq.async_find_device()
 
+    async def async_clean_room(self, **kwargs):
```

Comment:
There is already partial coverage of vacuum platform. You should add some tests for the new service.